server/todo-api/types.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface HTTPError extends Error {
+  status?: number;
+  errors?: { message: string }[];
```

Comment:
es mejor usar la sintaxis de generics en Typescript, ya que es mas clara
```suggestion
  errors?: Array<{ message: string }>;
```